stripe.go

```diff
@@ -329,7 +346,7 @@ func (s *BackendConfiguration) Do(req *http.Request, v interface{}) error {
 	}
 
 	if s.LogLevel > 2 {
-		s.Logger.Printf("Stripe Response: %q\n", resBody)
+		s.Logger.Printf("Response: %s\n", string(resBody))
```

Comment:
Changed the `%q` to a `%s` with a `string(...)` cast because the `%q` produces lots of noisy escaping that's not really needed for anything and detrimental to legibility:

``` go
Stripe Response: \"{\\\\"id\\\": \\\"ch_*\\\",\
```